guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) For <strong>example,</strong> the below line will first scale a dataset using a <strong>scikit-learn</strong> scaler followed by applying a <strong>scikit-learn</strong> model to the scaled dataset</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>